### PR TITLE
Changes to formatting and instructions for pgmonitor installatio READMEs

### PR DESCRIPTION
**Pull request by jkatz**

### CODE REVIEW

**Comment by jkatz** [comment]
@keithf4 @sharmay Attached are updates to some of the formatting + a couple of changes / recommendations from the process of installing.

Thanks for reviewing!

File: exporter/README.md [hunk 1]
```diff
@@ -1,124 +1,241 @@
...
- * https://github.com/prometheus/node_exporter/releases
- * https://github.com/wrouesnel/postgres_exporter/releases
- * https://github.com/keithf4/pg_bloat_check
+### Installation on RHEL/CentOS 7
 
-All executables are expected to be in /usr/bin. A base node_exporter systemd file is expected to be in place already. An example one can be found here https://github.com/lest/prometheus-rpm/tree/master/node_exporter
+#### With RPM Packages
 
-The files contained in this repository are assumed to be installed in the following locations with the following names. A double hash is replaced by the two-digit major version of PostgreSQL you are running (ex: 95, 96, 10, etc).
+There are RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the [Crunchy Customer Portal](https://access.crunchydata.com/).
```
[the remark this answers is not in the document]
Comment (jkatz) [reply exporter/README.md]: @sharmay It's already public on https://www.crunchydata.com/

File: grafana/README.md [hunk 2]
```diff
@@ -4,32 +4,30 @@ The Grafana RPM Package can be downloaded and installed from https://grafana.com
...
-  * PostgreSQL.json
-  * PostgreSQLDetails.json
-  * BloatDetails.json
-  * CRUD_Details.json
-  * TableSize_Details.json
-  * FilesystemDetails.json
+1. Connect to Grafana via https://&lt;ip-address&gt;:3000
+1. Login as admin/admin
+1. Change admin password
+1. Add a Prometheus datasource. Ensure the resource matches what you setup for pgmonitor, e.g. `localhost:9000`
```
[the remark this answers is not in the document]
Comment (jkatz) [reply grafana/README.md]: @sharmay You're right it's 9090, I will update.

File: grafana/README.md [hunk 3]
```diff
@@ -4,32 +4,30 @@ The Grafana RPM Package can be downloaded and installed from https://grafana.com
...
+  - [FilesystemDetails.json](FilesystemDetails.json)
 
 
 ### API Import
 
-It is possible to import these graphs through the "import" HTTP API using the following curl command to add some required wrapper information to each json blob. 
-```
+It is possible to import these graphs through the "import" HTTP API using the following curl command to add some required wrapper information to each json blob.
+
+```bash
```
[the remark this answers is not in the document]
Comment (jkatz) [reply grafana/README.md]: @sharmay I don't follow.

**Comment by jkatz** [comment]
@sharmay Thanks for the review! I incorporated the feedback and pushed another commit.

File: exporter/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sharmay) [reply exporter/README.md]: ok

File: grafana/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sharmay) [reply grafana/README.md]: curl --user username:password -S "http://localhost:3000/api/dashboards/import" -X POST -H 'Content-Type: application/json;charset=UTF-8' --data-binary "{  \"dashboard\" : $(cat PostgreSQL.json) , \"overwrite\":true, \"inputs\":[  {  \"name\":\"DS_PROMETHEUS\", \"type\":\"datasource\", \"pluginId\":\"prometheus\", \"value\":\"PROMETHEUS\" } ] }"

In above command \"value\":\"PROMETHEUS\"  has to match with datasource name they have created.

[the remark this answers is not in the document]
Comment (jkatz) [reply grafana/README.md]: @sharmay Ah I believe that is already here:

https://github.com/CrunchyData/pgmonitor/pull/46/files#diff-43fa532608e63d217a6d917cb0b26e3dR33

[the remark this answers is not in the document]
Comment (keithf4) [reply grafana/README.md]: That is already there as the final bullet listing the things that need to be changed by the user.

**Comment by jkatz** [comment]
@sharmay @keithf4 Please let me know if this needs any more changes. Thanks!

File: prometheus/README.md [hunk 4]
```diff
@@ -1,55 +1,156 @@
...
+
+For installations without using packages, you can download [Prometheus](https://prometheus.io/) and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) from [https://prometheus.io/download](https://prometheus.io/download).
+
+##### Minimum Versions
+
+|              | Minimum Version |
+|--------------|-----------------|
+| Prometheus   | 2.0             |
+| Alertmanager | 0.12.0          |
+
```
Comment (keithf4) [inline prometheus/README.md]: I would prefer not to list specific version names in the readme like this. It's very easy to forget to update it. Can mention that our build assumes Prometheus 2.x and should always use the latest minor versions of both Prometheus and the Alertmanager.

File: prometheus/README.md [hunk 5]
```diff
@@ -1,55 +1,156 @@
...
+| File                                     | Instructions |
+|------------------------------------------|--------------|
+| `/etc/prometheus/crunchy-prometheus.yml` | Modify to set scrape interval if different from the default of 30s. Activate alert rules and Alertmanager by uncommenting lines when set as needed. Default service expects config file to be named `crunchy-prometheus.yml` |
+| `/etc/prometheus/crunchy-alertmanager.yml` | Setup alert target (e.g., SMTP, SMS, etc.), receiver and route information. Default service expects config file to be named `crunchy-alertmanager.yml` |
+| `/etc/prometheus/crunchy-alert-rules.yml` | Update rules as needed. Default Prometheus config expects file to be named `crunchy-alert-rules.yml` |
+| `/etc/prometheus/auto.d/*.yml` | You will need at least one file with a `.yml` extension in this folder in order to autodiscover and scrape data from your [exporters](exporter/README.md). Copy the example file to create as many additional targets as needed.  Ensure the configuration files you want to use do not end in `.yml.example` but only with `.yml`. |
+
```
Comment (keithf4) [inline prometheus/README.md]: Last sentence for auto.d instructions seem redundant. Maybe reword the first sentence to say "You will need at least one file with a final `.yml` extension"

File: prometheus/README.md (hunk 4 above)
Comment (keithf4) [reply prometheus/README.md]: English doesn't seem coherent here. I would just say "Prometheus 2.x" and not mention any version of Alertmanager at all.

Comment (jkatz) [reply prometheus/README.md]: @keithf4 Updated.
